src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/ActionPlanSession.kt

```diff
@@ -19,7 +19,7 @@ import javax.validation.constraints.NotNull
 @Entity
 @Table(name = "action_plan_appointment")
 @TypeDef(name = "attended", typeClass = PostgreSQLEnumType::class)
-data class ActionPlanAppointment(
+data class ActionPlanSession(
```

Comment:
SessionAppointment?

Reply:
we talked about this quite a bit and after enumerating all the options decided on this.

we wanted to reservce 'appointment' for only the appointment entity itself. this way reducing confusion about which objects are appointments and which aren't.